reference/5.1/Microsoft.PowerShell.Core/About/about_Execution_Policies.md

```diff
@@ -86,7 +86,8 @@ The PowerShell execution policies are as follows:
 
 - There is no execution policy set in the current scope.
 - If the execution policy in all scopes is **Undefined**, the effective
-  execution policy is **Restricted**, which is the default execution policy.
+  execution policy is the **Restricted** for Windows Clients, and
```

Comment:
Remove the word `the` and remove the comma. Also, 'Clients' should be lowercase.